### PR TITLE
fix(owners): helia-docker

### PR DESCRIPTION
**Pull request by whizzzkid**
### Why do you need this?

The ownership was transferred over from: https://github.com/whizzzkid/helia-docker

I need to protect the main branch,

### What else do we need to know?

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

File: github/ipfs.yml [hunk 1]
```diff
@@ -5048,13 +5048,16 @@ repositories:
     visibility: public
   helia-docker:
     advanced_security: false
-    allow_update_branch: false
+    allow_update_branch: true
     archived: false
     collaborators:
+      admin:
+        - whizzzkid
```
Comment (SgtPooki) [inline github/ipfs.yml:5055]:
```suggestion
        - @ipfs/helia-dev 
```

?

**Review by SgtPooki — APPROVED** [review]
Im good with this but we should give admin to helia-dev entirely

File: github/ipfs.yml [hunk 2]
```diff
@@ -5048,13 +5048,16 @@ repositories:
     visibility: public
   helia-docker:
     advanced_security: false
-    allow_update_branch: false
+    allow_update_branch: true
     archived: false
     collaborators:
+      admin:
+        - helia-dev
```
Comment (galargh) [inline github/ipfs.yml:5055]: This is the collaborators part, it should be in the teams instead.
```suggestion
```

File: github/ipfs.yml [hunk 3]
```diff
@@ -5048,13 +5048,14 @@ repositories:
     visibility: public
   helia-docker:
     advanced_security: false
-    allow_update_branch: false
+    allow_update_branch: true
     archived: false
     collaborators:
```
Comment (galargh) [inline github/ipfs.yml:5053]: BTW, @whizzzkid, you can configure branch protections via GitHub Management like this even if you don't have admin access to a repo.

```suggestion
    branch_protection:
      main:
        allows_deletions: false
        allows_force_pushes: false
        blocks_creations: false
        enforce_admins: false
        lock_branch: false
        require_conversation_resolution: false
        require_signed_commits: false
        required_linear_history: false
        required_pull_request_reviews:
          dismiss_stale_reviews: false
          require_code_owner_reviews: false
          required_approving_review_count: 1
          restrict_dismissals: false
    collaborators:
```

**Pull request merged**